src/frontend/src/pages/GanttPage/GanttChartComponents/GanttTaskBar.tsx

```diff
@@ -182,8 +211,41 @@ const GanttTaskBar = ({
                 onMouseDown={handleMouseDown}
               />
             )}
+            {showPopup && (
+              <GanttToolTip
+                xCoordinate={cursorX}
+                yCoordinate={cursorY}
+                title={event.name}
+                startDate={event.start}
+                endDate={event.end}
+                color={event.styles?.backgroundColor}
+                projectLead={event.projectLead}
+                projectManager={event.projectManager}
+              />
+            )}
           </Box>
         </div>
+        {/* {event.children.map((child) => {
```

Comment:
Why is this commented out ?

Reply:
So if you try it and you uncomment that section, you will see the colors of the work package all in one line. My goal was to try to overlap that with the grey bar so that the main project bar has all of the colors of its corresponding work package but wasn't able to get that to work. Would you be able to give that a try and see if you can get it to work

Reply:
Did you get a chance to try this out?

Reply:
@Aaryan1203 make sure to send any update pings like this in slack, github notification are easy to miss bc there are so many of them

Reply:
leaving other comments first, then will try out